packages/frontend-shared/src/assets/icons/testing-type-component_x16.svg

```diff
@@ -0,0 +1,5 @@
+<svg data-cy="icon-testing-type-component" width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M2 10H6V14H2V10Z" fill="white" fill-opacity="0.5"/>
+  <path d="M10 2H14V6H10V2Z" fill="white" fill-opacity="0.5"/>
+  <path d="M2 6V5C1.44772 5 1 5.44772 1 6H2ZM6 6H7V5H6V6ZM2 14H1C1 14.5523 1.44772 15 2 15V14ZM10 2V1H9V2H10ZM14 2H15C15 1.44772 14.5523 1 14 1V2ZM14 6V7H15V6H14ZM10 6H9V7H10V6ZM10 10H11V9H10V10ZM10 14V15C10.5523 15 11 14.5523 11 14H10ZM2 7H6V5H2V7ZM5 6V10H7V6H5ZM6 9H2V11H6V9ZM3 10V6H1V10H3ZM5 10V14H7V10H5ZM6 13H2V15H6V13ZM3 14V10H1V14H3ZM10 3H14V1H10V3ZM13 2V6H15V2H13ZM14 5H10V7H14V5ZM11 6V2H9V6H11ZM6 11H10V9H6V11ZM9 10V14H11V10H9ZM10 13H6V15H10V13Z" fill="white"/>
+</svg>
```

Comment:
Random q: where do the icons/markdown come from?

Reply:
From Figma. You can click on the icon and export it as an svg. There is also the design-system page that has all the icons as well

https://www.figma.com/file/1WJ3GVQyMV5e7xVxPg3yID/Design-System?node-id=2448%3A3902

Reply:
Right, ok! I thought it was coming from some library or plugin, for some reason. This makes sense now.